pom.xml

```diff
@@ -68,7 +68,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <java.level>8</java.level>
     <jenkins.version>2.249.1</jenkins.version>
-    <jackson.version>${revision}</jackson.version>
+    <jackson.version>2.13.0</jackson.version>
```

Comment:
well this is not great...

either 1, I need to replace all the `versions` with `$revision` in the pom, or drop this and use the jackson pom... 🤷